templates/search_clients_daily_v7.sql

```diff
@@ -0,0 +1,153 @@
+WITH
+  augmented AS (
+  SELECT
+    *,
+    ARRAY_CONCAT( ARRAY(
```

Comment:
let's make sure each of these arrays start on a new line, for easier reading:
(I made sure suggestion indentation matches bigquery console formatting)
```suggestion
    ARRAY_CONCAT( --
      ARRAY(
```